pom.xml

```diff
@@ -177,6 +175,7 @@
             <detectLinks>false</detectLinks>
             <detectOfflineLinks>false</detectOfflineLinks>
             <quiet>true</quiet>
+            <sourcepath>${project.basedir}/src/main/java</sourcepath>
```

Comment:
i think you should be able to reference that using sourceDirectory property as described here http://maven.apache.org/pom.html#Resources

Reply:
Great suggestion, works like a charm.